geniza/corpus/admin.py

```diff
@@ -462,13 +463,48 @@ def get_urls(self):
     actions = (export_to_csv,)
 
 
+@admin.register(DocumentNeedsReview)
+class DocumentNeedsReviewAdmin(DocumentAdmin):
+    ordering = ("needs_review",)
+
+    list_display = (
+        "id",
+        "needs_review",
+        "shelfmark",
+        "description",
+        "doctype",
+        "all_languages",
+        "last_modified",
+        "is_public",
+    )
+
+    search_fields = (
+        "fragments__shelfmark",
+        "tags__name",
+        "description",
+        "notes",
+        "needs_review",
+        "id",
+        "old_pgpids",
+        "needs_review",
+    )
+
+    def get_queryset(self, request):
+        # ?: We often use `needs_review__is_empty` but I got this error. Is that expected?
+        #      Unsupported lookup 'isempty' for TextField or join on the field not permitted.
+        return super().get_queryset(request).exclude(needs_review="")
```

Comment:
I don't see `isempty` as an available queryset filter; maybe you meant `isnull`? This is the filter I would expect you to use, since we can't search on null for text fields.

Reply:
Oh I was drawing from html query parameters and got confused: `?needs_review__isempty=0`
Sounds good!